### ISSUE TITLE
Generate the accuracy and loss figures

### ISSUE THREAD

**Issue by samahwaleed** [sovit-123/fasterrcnn-pytorch-training-pipeline#59]
Dear Sir,

Could you please generate the training loss and training accuracy in one figure?

Thank you

**Comment by sovit-123** [comment]
@samahwaleed 
Hello, combining the two graphs may not help much. Because in object detection, we mostly look at the mAP graph and not the mAP graph and training loss together.
The reason is that mAP is calculated on the validation set and the loss here is saved on the training set.

**Comment by samahwaleed** [comment]
Thank you for your kind response. I would be very thankful if you add it temporarily as I have to show this graph in the research paper.
